StereoKitC/tools/file_picker.cpp

```diff
@@ -427,6 +459,10 @@ void file_picker_update() {
 				fp_call_status = true;
 			}
 			ui_sameline();
+			if (ui_button("ListMode")) {
```

Comment:
I'm not sure I really liked the way this looked in the UI, can you play around with the location some, and see what else can be done for it? This should also probably be some kind of toggle rather than button.